robottelo/constants/__init__.py

```diff
@@ -16,9 +16,9 @@ class Colored(Box):
 
 
 # This should be updated after each version branch
-SATELLITE_VERSION = "6.15"
+SATELLITE_VERSION = "6.16"
 SATELLITE_OS_VERSION = "8"
```

Comment:
Or "9"  right ? :wink:

Reply:
Not yet confirmed I believe, anyways OS Version would be provided by CI (either 8 or 9) for executing tests in CI! This is been managed by conf files!

If it supports on RHEL 9 , we can switch this drop down version to 9.

Reply:
Could we use settings.robottelo.rhel_version here provided by CI instead of this constant, so this can be managed by configs for each job for 8/9?

Reply:
@Gauravtalreja1 Upon looking the usage of this variable it is being used in deciding(returning) capsule repo in `SatelliteCapsuleRepository` class of `repository_mixins.py` module.

We should deprecate the use of this variable and switch the url def to use rhel_version  conf settings.

Reply:
I dont block this PR on it as that change has to be cherrypicked back to all zBranches but this PR is not for cherrypicking

Reply:
agree @jyejare , we could start with master for this deprecation or else you could handle this in different PR which we can cherrypick to older branch, but it's no blocker